variation-commons-mongodb/src/test/java/uk/ac/ebi/eva/commons/mongodb/MongoVariantConversionTest.java

```diff
@@ -159,7 +159,8 @@ private BasicDBObject buildMongoVariantWithFiles() {
     }
 
     private BasicDBObject buildMongoBasicVariant() {
-        VariantWithSamplesAndAnnotation variant = new VariantWithSamplesAndAnnotation(CHROMOSOME, START, END, REFERENCE, ALTERNATE);
+        VariantWithSamplesAndAnnotation variant = new VariantWithSamplesAndAnnotation(CHROMOSOME, START, END, REFERENCE,
+                                                                                      ALTERNATE, null);
```

Comment:
So is this a required change? or can be reverted as well?

Reply:
The constructor for `VariantWithSamplesAndAnnotation` still requires the mainId so we need the last argument